src/test/java/com/znsio/sample/e2e/businessLayer/amazonsearch/AmazonCartBL.java

```diff
@@ -0,0 +1,46 @@
+package com.znsio.sample.e2e.businessLayer.amazonsearch;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.entities.Platform;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonCartScreen;
+import org.apache.log4j.Logger;
+import org.assertj.core.api.SoftAssertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AmazonCartBL {
+    private static final Logger LOGGER = Logger.getLogger(AmazonCartBL.class.getName());
+    private final TestExecutionContext context;
+    private final SoftAssertions softly;
```

Comment:
remove unused variables

Reply:
should i also remove currentPlatform and currentUserPersona ?